modules/planning/navi/common/local_path.cc

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+/**
+ * @file
+ * @brief This file provides the implementation of the class "LocalPath".
+ */
+#include <vector>
+
+#include "modules/planning/navi/common/local_path.h"
+
+namespace apollo {
+namespace planning {
+
+bool LocalPath::GetInitY(double &y) {
+  if (points_.size() < 1) {
+    return false;
+  }
+  y = points_[0].y();
+  return true;
+}
+
+std::vector<Vec2d> LocalPath::GetXYVec() { return points_; }
+
+int LocalPath::GetRange() { return static_cast<int>(path_points_.size() - 1); }
+
+void LocalPath::Shift(double dist) {
+  for (int i = 0; i < path_points_.size(); ++i) {
+    path_points_[i].set_y(path_points_[i].y() + dist);
+  }
+}
+
+void LocalPath::Cut(double dist) { return; }
```

Comment:
function empty ?

Reply:
referring to local_path.py,delete this function?